ha/ham.c

```diff
@@ -230,12 +229,12 @@ static void ham_rpc_ctx_init(struct ham_rpc_ctx *ctx,
 			     const struct m0_fid *local_process)
 {
 	enum { NR_TMS = 1 };
-	int rc;
+	int                 rc;
```

Comment:
No need to change the alignment of this variable.

Reply:
incorporated